spec/features/authentication_tokens/revoke_authentication_token_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "revoke an authentication token" do
+  scenario "when the feature flag is on I can revoke a token", feature_token_management: true do
+    given_i_am_authenticated
+    and_i_have_generated_a_token
+
+    given_i_navigate_to_the_authentication_token_index_page
+    and_i_click_the_revoke_token_action
+    then_i_should_see_the_revoke_token_page
+
+    and_i_click_the_revoke_token_confirmation_button
+    then_the_token_should_have_a_revoked_status
+    and_i_should_no_longer_see_the_revoke_action
+    and_i_should_see_when_the_token_was_revoked
+  end
```

Comment:
I suggest to add scenarios per user role similar to [create_new_authentication_token_spec.rb](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/spec/features/authentication_tokens/create_new_authentication_token_spec.rb)

Reply:
Thinking about this, I think it's redundant as you can only really get to the revoke action if you can also generate a token.